specs/protocol/jovian/system-config.md

```diff
@@ -0,0 +1,32 @@
+# Jovian: System Config
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Minimum Base Fee Configuration](#minimum-base-fee-configuration)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Minimum Base Fee Configuration
+
+Jovian adds a new configuration value to `SystemConfig` to control the minimum base fee used by the EIP-1559 fee market
+on OP Stack chains. The value represents the log2 of the minimum base fee in wei.
+
+| Name             | Type    | Default | Meaning                                                                                 |
+|------------------|---------|---------|-----------------------------------------------------------------------------------------|
+| `minBaseFeeLog2` | `uint8` | `0`     | Log2 of the minimum base fee in wei. `0` disables the minimum base fee entirely.        |
```

Comment:
When we say `0` disables the feature, we should be more precise about how that is implemented. For example, this magic value can be totally opaque to the smart contracts layer, and it is only the _execution clients_ that need to detect it as a special case. Of course, the smart contract setter is the user interface to the feature so it needs to document the meaning of the `0` value. But in terms of specification, let's be clear that it is only downstream code that does the disabling.

Reply:
Does the Execution Client even need to special-case this? An enforced minimum of 0 seems to be a no-op that doesn't need any branched logic.

We could just say that "At 0, this feature is effectively disabled, because all fees are higher than 0"